styles/components/navigation.css

```diff
@@ -0,0 +1,11 @@
+.navigation {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 336px;
+  height: 55px;
```

Comment:
Please don't set a width – this should be responsive. Also: Don't set the height. It can get its height from its content or from the containing grid.